spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -2860,7 +2860,11 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
           case RightOuter => JoinType.RightOuter
           case FullOuter => JoinType.FullOuter
           case LeftSemi => JoinType.LeftSemi
-          case LeftAnti => JoinType.LeftAnti
+          // TODO: DF SMJ with join condition fails TPCH q21
```

Comment:
let me have a look on q21. I remember Anti join had issues with TPCH in DF but it was fixed

Reply:
But it might be also related to https://github.com/apache/datafusion/issues/11555